DxDispatch/doc/Guide.md

```diff
@@ -334,6 +350,71 @@ void CSMain(uint3 dtid : SV_DispatchThreadID)
 - You may declare shader resources using any type of buffer view, but textures are not supported. Arrays of resources (e.g. `Buffer<float> inputs[2];`), including unbounded arrays, are not yet supported. This is on the backlog though!
 - If you declare a resource in HLSL but do not reference it in the shader program then it will likely be optimized away! Binding failures will result if you try to bind a buffer in the model to an unused shader input.
 
+### ONNX Model
+
+You can execute ONNX models using ONNX Runtime with the DirectML execution provider using an ONNX dispatchable. The example below shows how to reference an ONNX model in the JSON model.
+
+```json
+{
+    "resources": 
+    {
+        "A0": { "initialValuesDataType": "FLOAT32", "initialValues": [ 1, 2, 3, 4, 5, 6 ] },
+        "B0": { "initialValuesDataType": "FLOAT32", "initialValues": [ 1, 2, 3, 4, 5, 6, 7, 8 ] },
+        "A1": { "initialValuesDataType": "FLOAT32", "initialValues": [ 1, 2, 3, 4, 5, 6 ] },
+        "B1": { "initialValuesDataType": "FLOAT32", "initialValues": [ 1, 2, 3, 4, 5, 6, 7, 8 ] },
```

Comment:
>"B1": { "initialValuesDataType": "FLOAT32", "initialValues": [ 1, 2, 3, 4, 5, 6, 7, 8 ] }, [](http://example.com/codeflow?start=8&length=90)

Question - if you don't pass a value for a graph input, does DxDispatch bind a default value to it like all zeros? (I found it convenient for running a model, when I didn't really care about the result, just the execution of it)

Reply:
Never mind - you say below.